Assets/Global/Scripts/Enemies/FollowEnemy.cs

```diff
@@ -43,15 +47,15 @@ private void Update()
         };
     }
 
-    public void ChangeState(FollowingEnemy.BaseState state)
+    public void ChangeState(FollowEnemyStates.StateBase state)
     {
-        if (currentState != null) currentState.Exit();
+        currentState?.Exit();
```

Comment:
if null check can be a single character :P, the `?`

Reply:
yeah i liked the verbosity / look of the if statement more in this case. but im fine with this as well.

Reply:
It doesnt matter for me, both work.
I just thought it was a cool little trick that can be usefull anywhere.

Since you can stack this questionmake aswell like: 

`int myVar = Something?.SomethingElse()?.MoreStuff ?? 1;` 
```cs 
int myVar = 1;
If(Something != null)
{
   If(Something.SomethingElse() != null) 
   {
       If(Something.SomethingElse().MoreStuff != null)
       {
           int myVar = Something.SomethingElse().MoreStuff ;
           }
       }
}```